armory/scenarios/scenario.py

```diff
@@ -374,33 +373,35 @@ def evaluate(self):
                 log.exception("Encountered error during scenario evaluation.")
             sys.exit(1)
 
-        if results is None:
+        if self.results is None:
```

Comment:
Actually, if `_evaluate` never sets `results`, then this raise an AttributeError.

This could be fixed by adding `self.results = None` in the scenario `__init__` method or changing this to something like `if getattr(self, "results", None) is None